src/v2/App.jsx

```diff
@@ -14,8 +14,25 @@
 // limitations under the License.
 // -----------------------------------------------------------------------------
 
-import React from 'react'
+import React, {useEffect} from 'react'
+
+import AppFooter from './components/AppFooter.jsx'
+import PotentialFireList from './components/PotentialFireList.jsx'
+
+import './App.css'
 
 export default function App() {
-  return <div/>
+  // XXX: Reset scroll position on page load. Otherwise, the window may be
+  // scrolled a couple hundred pixels down (not sure why).
+  useEffect(() => {
+    ;(function check() {
+      /complete/.test(document.readyState) ? window.scrollTo(0, 0) : setTimeout(check)
```

Comment:
Nope, the delay parameter is optional. If you leave it out it means “not now in this event loop, but as soon as possible in the next one” (which is equivalent to specifying `0`, IIRC). Using that here because we want the page to load and be in an acceptable state ASAP. I can be explicit if that’s better for you, but omitting it is a pretty standard convention/shorthand in my experience.